### PR TITLE
Master is now 24

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

**Review by skjnldsv — APPROVED** [review]

File: version.php [hunk 1]
```diff
@@ -30,15 +30,15 @@
 // between betas, final and RCs. This is _not_ the public version number. Reset minor/patchlevel
 // when updating major/minor version number.
 
-$OC_Version = [23, 0, 0, 7];
+$OC_Version = [24, 0, 0, 0];
 
 // The human readable string
-$OC_VersionString = '23.0.0 RC1';
+$OC_VersionString = '24.0.0 dev';
```
Comment (skjnldsv) [inline version.php:36]: usually we put `alpha` I think

Comment (nickvergessen) [reply version.php:36]: Yes we did in the past, but alpha is usually a packaged version (outside of nextcloud) so I went with dev what is the international way to do it.

**Comment by nickvergessen** [comment]
Core/Command/ patch is not needed anymore since @kesselb did fix that: https://github.com/nextcloud/server/pull/29027

**Comment by nickvergessen** [comment]
Failing integration tests will heal once Talk has 24 support on master (which will come once this is in, circular dependency problem)
```
/drone/src/build/integration/sharing_features/sharing-v1-video-verification.feature:471
```

File: version.php (hunk 1 above)
Comment (PVince81) [reply version.php:36]: voting for "dev" for now until we do a release

**Review by PVince81 — APPROVED** [review]
:+1:

**Review by juliusknorr — APPROVED** [review]

**Comment by nickvergessen** [comment]
Todo on old version: https://github.com/nextcloud/server/pull/29673

**Pull request merged**